lms/static/js/bootstrap-transition.js

```diff
@@ -22,7 +22,7 @@
 
   $(function () {
 
-    "use strict"; // jshint ;_;
+    "use strict";
```

Comment:
I think this passed because it's technically not a string, or a variable of any kind - it's an engine directive. Or it might be a bug in their parser? Strange either way.